architecture/2024-08-bicepdeployment-controller.md

```diff
@@ -0,0 +1,550 @@
+# Radius BicepDeployment Controller
+
+* **Author**: Will Smith (@willdavsmith)
+
+## Overview
+
+<!--
+Provide a succinct high-level description of the component or feature and 
+where/how it fits in the big picture. The overview should be one to three 
+paragraphs long and should be understandable by someone outside the Radius
+team. Do not provide the design details in this, section - there is a
+dedicated section for that later in the document.
+-->
+
+Today, users of Radius and future adopters of Radius use many Kubernetes-specific tools in their production workflows. Most of these tools operate on Kubernetes resources exclusively - which presents a problem when trying to deploy resources defined in Bicep manifests. This design proposes the creation of a new Kubernetes controller (BicepDeployment controller) in Radius that will allow users to deploy resources defined in Bicep manifests using Kubernetes tooling.
+
+## Terms and definitions
+
+<!--
+Include any terms, definitions, or acronyms that are used in
+this design document to assist the reader. They may or may not
+be part of the user-facing experience once implemented, and can
+be specific to this design context.
+-->
+
+**CRD (Custom Resource Definition)**: A definition for a Kubernetes resource that allows users to define their own resource types.
+
+**CR (Custom Resource)**: An instance of a CRD that represents a custom resource in Kubernetes.
+
+**Bicep**: A Domain Specific Language (DSL) for defining Azure resources, or when used with Radius, Azure resources, AWS resources, and Radius resources.
```

Comment:
```suggestion
**Bicep**: An infrastructure-as-code language for defining Azure resources, or when used with Radius, Azure resources, AWS resources, and Radius resources.
```

It's not-not a DSL 👍 IaC is more specific and the term we usually use. 

I think you could also simplify a big and remove the "for defining Azure resources" part, it's not super relevant.